releng/com.zeligsoft.papyrus-cx.axcioma.targetdefinition/papyrus-cx-axcioma-release.tpd

```diff
@@ -59,9 +61,18 @@ location "https://download.eclipse.org/releases/2020-06/" eclipse-2020-06 {
 	org.eclipse.tools.templates.freemarker	[1.1.100,2.0.0)
 	org.eclipse.tools.templates.ui			[1.1.100,2.0.0)
 	
+	org.eclipse.wst.common_ui.feature.feature.group	[3.11.0,4.0.0)
+	org.eclipse.wst.dtd.core
+	org.eclipse.wst.dtd.ui
+	org.eclipse.wst.dtd.ui.infopop
+	org.eclipse.wst.dtdeditor.doc.user
+	org.eclipse.wst.xml_ui.feature.feature.group	[3.18.0,4.0.0)
```

Comment:
Again, I'm no maven expert so maybe the rest of that `org.eclipse.wst...` stuff is needed but I think this is the only feature group we care about from that update site. Just letting you know in case you want to keep bloat to a minimum. Obviously, it's your decision about what to package in your product, I'm just letting you know the list can be slimmed down to just this feature group if needed.

Reply:
I agree that we need all required dependencies packaged as well.

I only mentioned the single feature group because that's all we installed before - but maybe it was pulling those other deps behind the scenes since we had access to them in the .zip of that update site (likely the case). I'll defer to you! If those are dependencies of the single feature group we are requesting then I reckon we need those pre-installed as well. Thanks @eposse !

Reply:
Just to try it, I removed the extra dependencies, disconnected from the internet and did a clean build, including only the main feature. It seems to work, and it automatically included the required plugins (although not the required features). But installing it and running it seems to work fine. The only caveat is that I'm not sure if the build worked because maven had previously fetched the dependencies on my machine or not. 

So you could try building offline the latest commit and test it to see if it works for you, and if so, we can merge it. If not, I'll add back the required dependencies.

Reply:
FYI - You can send `-D maven.repo.local <some_path>` into your `maven clean verify` call in order to change your m2 directory and force maven to use that `<some_path>` as its cache to test that those deps weren't already on your system.

If I try and build offline I think it'll fail to find all the tycho stuff needed so that won't be an option for me to test unfortunately. We build Java plugins with access to maven repos but install them on systems without internet access which is why I was saying all deps need to be installed alongside the plugins.

Reply:
Oh, I forgot that you do build online and it's the installation that is offline. In that case, then I think it's safe. Those bundles are fetched during the build and are included in the generated zip and the rpm. So let me know if you want to test this PR before merging it.

Reply:
I just installed the RPM generated by your GitHub Actions job and the XML editor isn't available. I searched for `wst` in the /opt install tree and got the following:
![image](https://user-images.githubusercontent.com/46937599/172846366-41db5131-efd0-4aa6-8bfe-16c4d0e075e1.png)

Notice there are no feature groups installed and I see no `org.eclipse.wst.xml.ui` plugin or feature listed.

Reply:
Hmm.. That is weird. When I run rpmbuild manually it does generate the full artifact. 

It is weird that so few plugins where included. Did you check the 'features' folder as well?

Also, can you try installing the rpm and do

```bash
rpm -qlp papyrus-cx-axcioma | grep xml
```
?

Reply:
My `find` command from the prior screenshot was searching all of the /opt install folder so yes both `plugins/` and `features/`.

After running the `rpm -ql papyrus-cx-axcioma | grep wst` (I simplified it from `xml` to `wst` as that's really what we care about) I get:
![image](https://user-images.githubusercontent.com/46937599/172877610-b4bb9447-c6f1-44e1-934e-f98c73e74be4.png)

The whole `xml` search gives:
```
/opt/papyrus-cx-axcioma/artifacts.xml
/opt/papyrus-cx-axcioma/configuration/org.eclipse.update/platform.xml
/opt/papyrus-cx-axcioma/features/com.zeligsoft.base_feature_4.0.0.202206061952/feature.xml
/opt/papyrus-cx-axcioma/features/com.zeligsoft.cx_feature_4.0.0.202206061952/feature.xml
/opt/papyrus-cx-axcioma/features/com.zeligsoft.domain.idl3plus_feature_4.0.0.202206061952/feature.xml
/opt/papyrus-cx-axcioma/features/com.zeligsoft.domain.ngc.ccm.axcioma_feature_4.0.0.202206061952/feature.xml
/opt/papyrus-cx-axcioma/features/com.zeligsoft.domain.omg.ccm_feature_4.0.0.202206061952/feature.xml
/opt/papyrus-cx-axcioma/features/com.zeligsoft.papyrus-cx.axcioma.rcp.feature_2.2.0.202206061952/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gdb.source_9.11.1.202004012021/META-INF/maven/org.eclipse.cdt.features/org.eclipse.cdt.gdb/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gdb.source_9.11.1.202004012021/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gdb_9.11.1.202004012021/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.build.source_9.11.1.202004012021/META-INF/maven/org.eclipse.cdt/org.eclipse.cdt.gnu.build/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.build.source_9.11.1.202004012021/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.build_9.11.1.202004012021/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.debug.source_9.11.1.202004012021/META-INF/maven/org.eclipse.cdt/org.eclipse.cdt.gnu.debug/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.debug.source_9.11.1.202004012021/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.debug_9.11.1.202004012021/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.dsf.source_9.11.1.202004012021/META-INF/maven/org.eclipse.cdt/org.eclipse.cdt.gnu.dsf/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.dsf.source_9.11.1.202004012021/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.gnu.dsf_9.11.1.202004012021/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.native.source_9.11.1.202005051837/META-INF/maven/org.eclipse.cdt/org.eclipse.cdt.native/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.native.source_9.11.1.202005051837/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.native_9.11.1.202005051837/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.platform.source_9.11.1.202006011430/META-INF/maven/org.eclipse.cdt/org.eclipse.cdt.platform/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.platform.source_9.11.1.202006011430/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.platform_9.11.1.202006011430/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.sdk_9.11.1.202006011430/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.source_9.11.1.202006011430/META-INF/maven/org.eclipse.cdt.features/org.eclipse.cdt/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt.source_9.11.1.202006011430/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.cdt_9.11.1.202006011430/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.draw2d.sdk_3.10.100.201606061308/META-INF/maven/org.eclipse.draw2d.features/org.eclipse.draw2d.sdk/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.draw2d.sdk_3.10.100.201606061308/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.draw2d.source_3.10.100.201606061308/META-INF/maven/org.eclipse.draw2d.features/org.eclipse.draw2d.source/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.draw2d.source_3.10.100.201606061308/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.draw2d_3.10.100.201606061308/META-INF/maven/org.eclipse.draw2d.features/org.eclipse.draw2d/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.draw2d_3.10.100.201606061308/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.e4.rcp_4.16.0.v20200604-0951/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.ecf.core.feature_1.5.600.v20200317-1602/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.ecf.core.ssl.feature_1.1.400.v20200317-1602/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.ecf.filetransfer.feature_3.14.800.v20200317-1602/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.ecf.filetransfer.httpclient45.feature_1.0.400.v20200522-1203/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.ecf.filetransfer.ssl.feature_1.1.300.v20200317-1602/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.egit_5.8.0.202006091008-r/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ecore.source_2.22.0.v20200324-0947/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.codegen.ecore/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ecore.source_2.22.0.v20200324-0947/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ecore.ui.source_2.22.0.v20200324-0947/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.codegen.ecore.ui/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ecore.ui.source_2.22.0.v20200324-0947/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ecore.ui_2.22.0.v20200324-0947/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ecore_2.22.0.v20200324-0947/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.source_2.20.0.v20200324-0932/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.codegen/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.source_2.20.0.v20200324-0932/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ui.source_2.22.0.v20200424-0451/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.codegen.ui/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ui.source_2.22.0.v20200424-0451/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen.ui_2.22.0.v20200424-0451/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.codegen_2.20.0.v20200324-0932/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.common.source_2.19.0.v20200324-0932/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.common/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.common.source_2.19.0.v20200324-0932/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.common.ui.source_2.17.0.v20190507-0402/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.common.ui/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.common.ui.source_2.17.0.v20190507-0402/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.common.ui_2.17.0.v20190507-0402/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.common_2.19.0.v20200324-0932/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.compare.diagram.gmf_3.3.11.202005260905/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.compare.egit_3.3.11.202005260905/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.compare.ide.ui_3.3.11.202005260905/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.compare.rcp.ui_3.3.11.202005260905/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.compare.uml2_3.3.11.202005260905/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.compare_3.3.11.202005260905/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.converter.source_2.15.0.v20200324-0723/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.converter/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.converter.source_2.15.0.v20200324-0723/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.converter_2.15.0.v20200324-0723/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.databinding.edit.source_1.7.0.v20190323-1059/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.databinding.edit/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.databinding.edit.source_1.7.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.databinding.edit_1.7.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.databinding.source_1.7.0.v20190323-1059/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.databinding/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.databinding.source_1.7.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.databinding_1.7.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.doc_2.21.0.v20200324-0723/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore.edit.source_2.14.0.v20190822-1451/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.ecore.edit/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore.edit.source_2.14.0.v20190822-1451/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore.edit_2.14.0.v20190822-1451/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore.editor.source_2.17.0.v20190528-0725/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.ecore.editor/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore.editor.source_2.17.0.v20190528-0725/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore.editor_2.17.0.v20190528-0725/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore.source_2.22.0.v20200519-1135/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.ecore/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore.source_2.22.0.v20200519-1135/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.ecore_2.22.0.v20200519-1135/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.edit.source_2.16.0.v20190920-0401/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.edit/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.edit.source_2.16.0.v20190920-0401/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.edit.ui.source_2.19.0.v20200205-0529/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.edit.ui/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.edit.ui.source_2.19.0.v20200205-0529/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.edit.ui_2.19.0.v20200205-0529/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.edit_2.16.0.v20190920-0401/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ecore.editor.source_2.14.0.v20190528-0725/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.mapping.ecore.editor/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ecore.editor.source_2.14.0.v20190528-0725/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ecore.editor_2.14.0.v20190528-0725/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ecore.source_2.12.0.v20190323-1059/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.mapping.ecore/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ecore.source_2.12.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ecore_2.12.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.source_2.13.0.v20190323-1059/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.mapping/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.source_2.13.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ui.source_2.13.0.v20190323-1059/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf.mapping.ui/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ui.source_2.13.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping.ui_2.13.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mapping_2.13.0.v20190323-1059/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mwe.core.feature_1.5.3.v20200520-0756/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mwe.ui.feature_1.5.3.v20200520-0756/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mwe2.language.sdk_2.11.3.v20200520-0756/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mwe2.launcher.source_2.11.3.v20200520-0756/META-INF/maven/org.eclipse.emf/org.eclipse.emf.mwe2.launcher/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mwe2.launcher.source_2.11.3.v20200520-0756/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mwe2.launcher_2.11.3.v20200520-0756/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.mwe2.runtime.sdk_2.11.3.v20200520-0756/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.query.doc_1.12.0.201805030653/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.query.ocl.source_1.12.0.201805030653/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.query.ocl_1.12.0.201805030653/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.query.sdk_1.12.0.201805030653/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.query.source_1.12.0.201805030653/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.query_1.12.0.201805030653/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.sdk_2.22.0.v20200519-1135/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.source_2.22.0.v20200519-1135/META-INF/maven/org.eclipse.emf.features/org.eclipse.emf/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.source_2.22.0.v20200519-1135/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.transaction.doc_1.12.0.201805140824/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.transaction.sdk_1.12.0.201805140824/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.transaction.source_1.12.0.201805140824/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.transaction_1.12.0.201805140824/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.validation.doc_1.12.1.201812070911/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.validation.ocl.source_1.12.1.201812070911/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.validation.ocl_1.12.1.201812070911/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.validation.sdk_1.12.1.201812070911/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.validation.source_1.12.1.201812070911/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.validation_1.12.1.201812070911/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.workspace.doc_1.12.0.201805140824/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.workspace.source_1.12.0.201805140824/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf.workspace_1.12.0.201805140824/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.emf_2.22.0.v20200519-1135/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.equinox.p2.core.feature_1.6.600.v20200511-1530/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.equinox.p2.extras.feature_1.4.800.v20200511-1530/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.equinox.p2.rcp.feature_1.4.800.v20200602-1138/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.equinox.p2.user.ui_2.4.800.v20200602-1138/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gef.sdk_3.11.0.201606061308/META-INF/maven/org.eclipse.gef.features/org.eclipse.gef.sdk/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gef.sdk_3.11.0.201606061308/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gef.source_3.11.0.201606061308/META-INF/maven/org.eclipse.gef.features/org.eclipse.gef.source/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gef.source_3.11.0.201606061308/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gef_3.11.0.201606061308/META-INF/maven/org.eclipse.gef.features/org.eclipse.gef/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gef_3.11.0.201606061308/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.examples.runtime.ui.pde_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.examples.runtime_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.runtime.notation.sdk_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.runtime.notation.source_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.runtime.notation_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.runtime.sdk_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.runtime.thirdparty.source_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.runtime.thirdparty_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf.source_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.gmf_1.13.0.202004160913/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.help_2.3.200.v20200604-0540/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.jgit.gpg.bc_5.8.0.202006091008-r/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.jgit.http.apache_5.8.0.202006091008-r/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.jgit.ssh.apache_5.8.0.202006091008-r/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.jgit.ssh.jsch_5.8.0.202006091008-r/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.jgit_5.8.0.202006091008-r/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.launchbar_2.4.100.202001141724/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.ocl.ui_2.12.0.v20200608-1555/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.ocl_3.14.3.v20200608-1555/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.compare.feature_0.7.0.202004302326/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.compare.feature_0.7.0.202004302326/pom.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.doc.feature_3.1.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.editor.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.emf.facet.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.emf.feature_1.1.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.architecture.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.core.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.emf.expressions.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.emf.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.gmfdiag.css.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.gmfdiag.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.internationalization.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.nattable.feature_4.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.services.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.ui.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.viewpoints.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.infra.widget.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.sdk.feature_4.8.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.alf.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.architecture.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.decoratormodel.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.diagram.css.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.diagram.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.expressions.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.feature_4.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.internationalization.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.m2m.qvto.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.modelexplorer.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.nattable.feature_4.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.properties.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.search.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.textedit.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.tools.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.ui.feature_1.2.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.uml.xtext.integration.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.views.documentation.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.views.feature_1.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.views.modelexplorer.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.views.properties.feature_3.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.views.references.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.views.search.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.papyrus.views.validation.feature_2.0.0.202006100749/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.platform_4.16.0.v20200604-0951/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.rcp_4.16.0.v20200604-0951/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2.codegen.ecore.ui_2.5.0.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2.common.edit_2.5.0.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2.common_2.5.0.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2.doc_5.5.0.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2.examples_5.5.0.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2.sdk_5.5.1.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2.uml.edit_5.5.0.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2.uml_5.5.0.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.uml2_5.5.1.v20200302-1312/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xpand.doc_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xpand.examples.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xpand.examples_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xpand.sdk_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xpand.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xpand.ui.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xpand.ui_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xpand_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.dependencies.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.dependencies_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.sdk_2.22.0.v20200602-1533/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.typesystem.emf.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.typesystem.emf_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.typesystem.uml2.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.typesystem.uml2_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.typesystem.xsd.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.typesystem.xsd_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.ui.source_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend.ui_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtend_2.2.0.v201605260315/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtext.runtime_2.22.0.v20200602-1533/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtext.ui_2.22.0.v20200602-1533/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtext.xbase.lib_2.22.0.v20200602-0715/feature.xml
/opt/papyrus-cx-axcioma/features/org.eclipse.xtext.xbase_2.22.0.v20200602-1533/feature.xml
/opt/papyrus-cx-axcioma/p2/org.eclipse.equinox.p2.core/cache/artifacts.xml
/opt/papyrus-cx-axcioma/plugins/com.sun.xml.bind_2.2.0.v201505121915.jar
/opt/papyrus-cx-axcioma/plugins/javax.xml.bind_2.2.0.v201105210648.jar
/opt/papyrus-cx-axcioma/plugins/javax.xml.stream_1.0.1.v201004272200.jar
/opt/papyrus-cx-axcioma/plugins/javax.xml_1.3.4.v201005080400.jar
/opt/papyrus-cx-axcioma/plugins/org.apache.ant_1.10.8.v20200515-1239/META-INF/maven/org.eclipse.orbit.bundles/org.apache.ant/pom.xml
/opt/papyrus-cx-axcioma/plugins/org.apache.batik.xml.source_1.6.0.v201011041432.jar
/opt/papyrus-cx-axcioma/plugins/org.apache.batik.xml_1.6.0.v201011041432.jar
/opt/papyrus-cx-axcioma/plugins/org.apache.xml.resolver_1.2.0.v201005080400.jar
/opt/papyrus-cx-axcioma/plugins/org.apache.xml.serializer_2.7.1.v201005080400.jar
/opt/papyrus-cx-axcioma/plugins/org.apache.xmlgraphics_2.3.0.v20190515-0436.jar
/opt/papyrus-cx-axcioma/plugins/org.eclipse.emf.mapping.ecore2xml.source_2.11.0.v20180706-1146.jar
/opt/papyrus-cx-axcioma/plugins/org.eclipse.emf.mapping.ecore2xml.ui.source_2.12.0.v20190528-0725.jar
/opt/papyrus-cx-axcioma/plugins/org.eclipse.emf.mapping.ecore2xml.ui_2.12.0.v20190528-0725.jar
/opt/papyrus-cx-axcioma/plugins/org.eclipse.emf.mapping.ecore2xml_2.11.0.v20180706-1146.jar
/opt/papyrus-cx-axcioma/plugins/org.eclipse.equinox.security.linux.x86_64_1.1.300.v20190830-1238/fragment.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.jdt.debug_3.15.100.v20200525-2012/plugin.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.nebula.widgets.richtext_1.2.0.201703081533/META-INF/maven/org.eclipse.nebula/org.eclipse.nebula.widgets.richtext/pom.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/data/21/fragment/fragment.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/data/21/plugin/plugin.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/data/30/fragment/fragment.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/data/30/plugin/plugin.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/plugin.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/scripts/build.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/scripts/genericTargets.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/scripts/package.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/scripts/productBuild/allElements.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/scripts/productBuild/productBuild.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/templates/features/customBuildCallbacks.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/templates/headless-build/allElements.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/templates/headless-build/customAssembly.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/templates/headless-build/customTargets.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/templates/packager/customTargets.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.pde.build_3.10.800.v20200410-1419/templates/plugins/customBuildCallbacks.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.platform_4.16.0.v20200604-0540/helpData.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.platform_4.16.0.v20200604-0540/intro/migrateExtensionContent.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.platform_4.16.0.v20200604-0540/intro/overviewExtensionContent.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.platform_4.16.0.v20200604-0540/intro/whatsnewExtensionContent1.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.platform_4.16.0.v20200604-0540/intro/whatsnewExtensionContent2.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.platform_4.16.0.v20200604-0540/intro/whatsnewExtensionContent3.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.platform_4.16.0.v20200604-0540/introData.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.platform_4.16.0.v20200604-0540/plugin.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.ui.themes_1.2.1000.v20200528-1125/plugin.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.wst.xml.core_1.2.200.v202005180122.jar
/opt/papyrus-cx-axcioma/plugins/org.eclipse.xpand.doc_2.2.0.v201605260315/help/index.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.xpand.doc_2.2.0.v201605260315/plugin.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.xpand.doc_2.2.0.v201605260315/toc.xml
/opt/papyrus-cx-axcioma/plugins/org.eclipse.xpand.examples_2.2.0.v201605260315/plugin.xml
```

Reply:
Hmm... Strange. I'm looking into it. Maybe we do need to explicitly package all the dependencies after all.

Reply:
Ok, something really odd is going on. I just checked the contents of both the RCP zip fila and RCP RPM produced by the last build of this pull request (I got them from here: [https://github.com/ZeligsoftDev/CX4CBDDS/actions/runs/2463004735](https://github.com/ZeligsoftDev/CX4CBDDS/actions/runs/2463004735)) and they both include all of the required WST XML bundles, including the main feature and all required features and plugins. Then I tried a clean install in a CentOS VM and succeeded and contains these features and plugins as well.

Can you confirm which is the exact build number that you tried?

Reply:
I downloaded `papyrus-cx-axcioma-2.2.0-202206081639.x86_64.rpm` from 
https://pipelines.actions.githubusercontent.com/serviceHosts/61881ef5-0739-4031-80cb-932447524941/_apis/pipelines/1/runs/186/signedartifactscontent?artifactName=papyrus-cx-axcioma-2.2.0-202206081639.x86_64.rpm&urlExpires=2022-06-09T12%3A10%3A15.5739197Z&urlSigningMethod=HMACV2&urlSignature=F77tu0EZ888i%2FsyBTvu%2B1bT%2BZaofj8GYo%2BOedLu6t10%3D

Reply:
Stranger still, because that's the same I downloaded and installed.

I'm rebuilding to double check. I also added a step to the GitHub actions workflow to print the GitHub context to ensure we are building the correct commit.

Reply:
I can confirm that the latest RPM generated by the GitHub build [here](https://github.com/ZeligsoftDev/CX4CBDDS/suites/6866341038/artifacts/265484996) does include all the XML features and plugins. Maybe it's something on your end?

Reply:
Merge away! - assuming your team is happy with the code of course.